### DISCUSSION TITLE
Necesito alguien que me guíe

### DISCUSSION THREAD

**Discussion by NoirOrange** [Launch-X-Latam/MisionFrontEnd#318] [Q&A]
Buen día, disculpen la molestia.

Debido a diversos problemas no he podido avanzar en Launch X, prácticamente no he visto nada.

Tengo entendido que tenemos que realizar prácticas tomando como guía los lives del grupo de Teams. El problema aquí es que aún no se me dá acceso al grupo (ya he envíado el correo correspondiente al link proporcionado).
-> [Entrar al grupo de Teams ](https://forms.office.com/pages/responsepage.aspx?id=BUvlSn63JEKu8YZhQi4IFqqReLKxwkROhdw1fyoxsStUNUZSMjhFUjRPNk03MUgxNFRMSklLTFJBVS4u) 

Entonces no me queda muy claro cómo se tienen que hacer los trabajos y todo esto me tiene muy estresada 😔

Por favor, si alguien me podría guíar, contáctame en facebook: https://www.facebook.com/hayley.oxton.7

**Comment by AlvaroFierro** [comment]
Hola @NoirOrange, me imagino que han cerrado el acceso al formulario, pero sugiero que te puedas comunicar con el equipo de @innovaccionvirtual por instagram o por correo a contacto@innovaccion.mx

**Reply by NoirOrange** [reply]
Llené el formulario desde el inicio de Launch X, y hasta ahora no he tenido acceso. Pero sí, intentaré comunicarme...

**Comment by RIvanCF** [comment]
Si gustas puedes contactarme y ahí darte una pequeña orientación para que puedas hacer tus practicas bien.

**Reply by NoirOrange** [reply]
Hola, por dónde puedo contactarte?? Me ayudarías muchísimo

**Reply by NoirOrange** [reply]
¿Hola? Por dónde podría contactarte? Aún espero tu respuesta.

**Reply by NoirOrange** [reply]
@RIvanCF No sé si te notifique, así que te etiqueto, por si acaso.

**Reply by RIvanCF** [reply]
Que onda amigo mio, te paso mi correo lx00959@innovaccion.mx

**Reply by NoirOrange** [reply]
@RIvanCF Ya te mandé correo, gracias

**Comment by doguedogue** [comment]
Checate los videos https://web.microsoftstream.com/browse

**Comment by Gilberto-Guzman** [comment]
Holi, te doy un resumen rápido de lo que hemos estado haciendo:
Referente a las clases, actualmente estamos finalizando el modulo 04 de JS. Pero no te preocupes, las clases están basadas en la documentación que se nos proporciono: https://github.com/Launch-X-Latam/MisionFrontEnd

Ahora bien respecto a las practicas, estas se muestran al final de cada modulo, y las tendrás que subir a tu repositorio personal para que puedan corroborar tu progreso, si quieres te puedes guiar de mi repo: https://github.com/Gilberto-Guzman/Launch-X

Cualquier duda no dudes en preguntarme :D

**Reply by Chemamx52** [reply]
Amigo, me podrías explicar cómo subir mis practicas a mi repositorio personal ? me seria de mucha ayuda ya que igual entre ayer a penas

**Reply by AlvaroFierro** [reply]
> Amigo, me podrías explicar cómo subir mis practicas a mi repositorio personal ? me seria de mucha ayuda ya que igual entre ayer a penas 

Checa la discusión #247 ese es un manual

**Reply by Gallardo1225** [reply]
Oye amigo. Tengo una duda con mi repositorio. Si lo tengo  que enlazar a LAUNCHX  para subirlos. Y si es así como lo hago. 
Esque en esa parte me confundí mucho . 
Gracias de ante mano .

**Reply by AlvaroFierro** [reply]
No, subelos a tu cuenta de GitHub, nadamas. 
Y en verdad, relajate, si  no comprendes algo, vuelve a leerlo y asi. 
No es mala onda, pero esta carrerar es de mucho leer, y está ahí mismo en la discusión #247

**Reply by NoirOrange** [reply]
@Gilberto-Guzman Te podría contactar para ver si me pudieras apoyar en todo esto?

**Comment by Gilberto-Guzman** [comment]
Hola @Chemamx52
Para subir las practicas al repositorio personal puedes utilizar la interfaz grafica que github ya te provee por defecto:
1.-
![image](https://user-images.githubusercontent.com/90573642/200187170-ad5bfd64-c48a-4129-b3b5-e8f3b193e573.png)
2.-
![image](https://user-images.githubusercontent.com/90573642/200187225-54cb049e-30cb-4032-b317-110f1e2574df.png)
3.-
![image](https://user-images.githubusercontent.com/90573642/200187244-ad51d131-8460-4280-a830-635436dbf14e.png)
4.-
![image](https://user-images.githubusercontent.com/90573642/200187410-c894f42f-16a6-4769-ae5d-070b2019500e.png)

Cualquier duda dime va?

**Comment by Gilberto-Guzman** [comment]
Hola @Gallardo1225 
Con que lo subas a tu github personal ya es mas que suficiente.

**Comment by Amhedriel** [comment]
Buenas noches!

Por mi parte colaboro con un tutorial de página [web gratis](https://github.com/Amhedriel/Tutoriales_Git_BASH_MarkDown/tree/master/Git/GitHub "Redirección a mí repositor") con GitHub, espero sea útil.

Saludos
